src/Console/Application.php

```diff
@@ -64,23 +59,17 @@ final class Application extends BaseApplication
 
     private const PACKAGE_NAME = 'infection/infection';
 
-    private const INFECTION_PREFIX = 'INFECTION';
-
     private const LOGO = '
     ____      ____          __  _
    /  _/___  / __/__  _____/ /_(_)___  ____
    / // __ \/ /_/ _ \/ ___/ __/ / __ \/ __ \
  _/ // / / / __/  __/ /__/ /_/ / /_/ / / / /
 /___/_/ /_/_/  \___/\___/\__/_/\____/_/ /_/
+
```

Comment:
is it needed?

Reply:
Before we were doing:

```php
$output->writeln([self::LOGO, $this->getLongVersion()]);
```

but now I do:

```php
return self::LOGO . parent::getHelp();
```

So either I add a `PHP_EOL` in-between above or add a line break in the logo. I don't mind either way so up to you